perfkitbenchmarker/providers/alicloud/util.py

```diff
@@ -137,7 +149,7 @@ def AddPubKeyToHost(host_ip, password, keyfile, username):
 
 
 def GeneratePassword(length=PASSWD_LEN):
-  digit_len = length / 2
+  digit_len = int(length / 2)
```

Comment:
Please use Python 3 style division `//` and `from __future__ import division`

Reply:
Done.